src/elexmodel/models/BaseElectionModel.py

```diff
@@ -264,3 +266,37 @@ def get_coefficients(self):
         These coefficients are for the point prediciton only, not for the lower or upper intervals models.
         """
         return self.features_to_coefficients
+
+    def find_optimal_lambda(
+        self,
+        reporting_units,
+        possible_lambda_values=[],
+        estimand="",
+        K=3,
+    ):
+        if len(possible_lambda_values) == 0:
+            return 0, None
+        if len(possible_lambda_values) == 1:
+            return possible_lambda_values[0], None
+
+        MAPE_arr = np.zeros_like(possible_lambda_values)  # array of MAPE sums for each lambda input
```

Comment:
`zeros_like` returns an array where the `dtype` is `int64` that means that all values will end up being cast an integers (so all values here are zero).